source/includes/starting-compass-paste-string.rst

```diff
@@ -1,6 +1,10 @@
 .. procedure:: 
    :style: normal 
 
+   .. step:: Open the :guilabel:`New Connection` modal.
```

Comment:
small thing but the numbering in this section looks a little off (in dark mode at least). The circles are right on the border of a shaded box. 
<img width="78" alt="Screenshot 2024-08-16 at 7 08 33 PM" src="https://github.com/user-attachments/assets/f4eca0f8-f0e6-4121-b559-28f2d59cfa94">

Reply:
I believe this is how produces render in dark mode - I can bring this up to the docs platform team and ask if this is supposed to be this way!

Reply:
sounds good! totally not blocking